test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesWithBasePathTest.cs

```diff
@@ -443,18 +444,20 @@ public async Task PageConventions_CustomizedModelCanPostToHandlers()
         public async Task PageConventions_CustomizedModelCanWorkWithModelState()
         {
             // Arrange
-            var getPage = await Client.GetAsync("/CustomModelTypeModel");
+            var getPage = await Client.GetAsync("/CustomModelTypeModel?Attempts=0");
```

Comment:
This isn't strictly necessary because the `OnGet(...)` handler doesn't check `ModelState.IsValid`.